src/app/onion/learning-object-builder/components/content-upload/app/services/file-management.service.ts

```diff
@@ -525,41 +526,4 @@ export class FileManagementService {
       }
     });
   }
-
-  /**
-   * Deletes specified from Learning Object using CLARK API
-   *
-   * @param {string} learningObjectID
-   * @param {string} filename
-   * @returns {Promise<{}>}
-   * @memberof FileManagementService
-   */
-  delete(learningObject: LearningObject, fileId: string): Promise<{}> {
-    const route = FILE_ROUTES.DELETE_FILE({
-      username: learningObject.author.username,
-      learningObjectId: learningObject.id,
-      fileId
-    });
```

Comment:
Was this not used?

Reply:
It was but we reorganized to have the routes stored alongside the service to imitate gateway. It looks like half of the services are empty files because we haven't been moving them over as we update the client.